### PR TITLE
added local install path for libraries and new run command function

### PR DESCRIPTION
**Pull request by riley206**
The run_command function allows you to pass a list of commands into the active environment. Using this you can add things to the config store, and much more.

### CODE REVIEW

File: src/volttrontesting/platformwrapper.py [hunk 1]
```diff
@@ -405,6 +405,33 @@ def get_agent_by_identity(self, identity):
             if agent.get('identity') == identity:
                 return agent
 
+    def run_command(self, command: list):
```
Comment (craig8) [inline src/volttrontesting/platformwrapper.py:408]: This is unnessary we already have an environment setup that has a run in it.

If you look at the install_agent it uses the appropriate environment and simply calls the run command.  Please try and have the same structure.

Comment (craig8) [reply src/volttrontesting/platformwrapper.py:408]: Or was this so that I could use it?

Comment (craig8) [reply src/volttrontesting/platformwrapper.py:408]: self._virtual_env.run(args=cmd, capture=True, cwd=self.volttron_home) from inside the platform wrapper does this exact same thing as this command.

**Review by craig8 — CHANGES_REQUESTED** [review]

File: src/volttrontesting/platformwrapper.py (hunk 1 above)
Comment (riley206) [reply src/volttrontesting/platformwrapper.py:408]: You told me to add this. This allows everyone to easily just run a command in their test.

Comment (craig8) [reply src/volttrontesting/platformwrapper.py:408]: my bad...I was talking about the install_config_entry function.

**Comment by craig8** [comment]
Follow on PR #53

**Pull request closed without merging**